src/collections/_documentation/platforms/javascript/angular.md

```diff
@@ -47,6 +105,48 @@ export class SentryErrorHandler implements ErrorHandler {
 export class AppModule {}
 ```
 
+When using your own `ErrorHandler`, make sure that whenever you use `HttpInterceptor` alongside it,
+it doesn't modify the error captured originally.
+The same goes for writing your own API Services with built-in `http` methods.
+
+For example, the service below make it impossible for the SDK to extract correct value.
```

Comment:
```suggestion
For example, the service below makes it impossible for the SDK to extract the correct value.
```

Reply:
And here, @MimiDumpling.

(Sorry, this is probably wicked confusing, since I'm working my way up rather than down...)